sdk/rust/src/http.rs

```diff
@@ -61,6 +61,30 @@ impl Request {
         builder
     }
 
+    /// Creates a [`RequestBuilder`] to fetch the headers from `uri`
+    pub fn head(uri: impl Into<String>) -> RequestBuilder {
+        RequestBuilder::new(Method::Head, uri)
+    }
```

Comment:
Head requests are pretty rare, is this worth the extra method? The work around is pretty straightforward.